src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertAdditionalServices.sql

```diff
@@ -163,7 +163,7 @@ BEGIN
 		    INSERT INTO dbo.CataloguePrice
 			    (CatalogueItemId, ProvisioningTypeId, CataloguePriceTypeId, PricingUnitId, TimeUnitId, CurrencyCode, LastUpdated, Price) 
 			    VALUES
-			    (@additionalServiceId, 1, 1, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 1, 'GBP', @now, 499.99);
+			    (@additionalServiceId, 3, 1, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', null, 'GBP', @now, 499.99);
```

Comment:
we would need a different price unit here as per patient is it's thing.

Reply:
Change to Per License